protected/views/element/update.php

```diff
@@ -144,60 +134,75 @@
 JS
 );
 
-
 $attributes = [];
 foreach ($model->colorAttributes() as $attribute) {
     $attributes[$attribute] = [
         'type' => Form::INPUT_HTML5,
         'html5type' => 'color',
-        'allowUnsafe' => true,
     ];
 }
 
+$this->registerCss(<<<CSS
+.columns .form-group {
+    break-inside: avoid;
+    margin-bottom: 0.25rem;
```

Comment:
I think these should go in the global `forms.css` CSS. I think we currently have no margin but that's not visible because the help-block as a margin. When the help block is contains an error message it the spacing is not optimal.